app/homepage/index.html

```diff
@@ -15,7 +15,7 @@ <h4 id="kennedyTimeNow">The current time at Kennedy Space Centre: </h4>
         </div>
         <img src="https://images-assets.nasa.gov/image/PIA22110/PIA22110~orig.jpg" alt="A photo of the Mars Rover">
         <h3>The latest views from the Mars Rover!</h3>
-        <a href="./login.html">Login to your NASA account.</a>
+        <a href="../login/">Login to your NASA account.</a>
```

Comment:
I _think_ URLs are normally written out relative to the base (i.e. relative to the home page "/") rather than relative to the current page.

So this would be `href="/login"`

Reply:
Yes, I can see that now. Thanks. :)